packages/core/core/src/services/entity-validator/validators.ts

```diff
@@ -80,8 +80,9 @@ const addMinIntegerValidator = (
     attr,
   }: {
     attr: Schema.Attribute.Integer | Schema.Attribute.BigInteger;
-  }
-) => (_.isNumber(attr.min) ? validator.min(_.toInteger(attr.min)) : validator);
+  },
+  { isDraft }: ValidatorOptions
+) => (_.isNumber(attr.min) && !isDraft ? validator.min(_.toInteger(attr.min)) : validator);
```

Comment:
Why don't you do the same isDraft check for addMaxIntegerValidator below?

Reply:
will add some comments explaining the behaviour 🙌 
only max validations will apply on drafts (for number and text fields)